maro/cli/data_pipeline/vm_scheduling.py

```diff
@@ -52,13 +52,7 @@ def __init__(self, topology: str, source: str, sample: int, seed: int, is_temp:
         self._cpu_readings_file_name_list = []
         self._clean_cpu_readings_file_name_list = []
 
-        self.aria2 = aria2p.API(
-            aria2p.Client(
-                host="http://localhost",
-                port=6800,
-                secret=""
-            )
-        )
+        self.aria2 = aria2p.API(aria2p.Client(host="http://localhost", port=6800, secret=""))
```

Comment:
Looks like it does not exceed the line length limitation. So what's the standard of breaking lines?